main.go

```diff
@@ -132,63 +132,98 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
-	milestones := []milestoneAPI{}
-
+func getActiveMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
 	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
 	URL := strings.Join(strURL, "")
+	milestones := map[string]string{}
 
-	m, linkHeader, err := callServer(URL, token, milestones)
+	m, err := gitlabGet(URL, token)
 	if err != nil {
-		return m, err
+		return milestones, err
```

Comment:
Maybe the function could return `nil, err` when there's an error to make sure that invalid data isn't passed on to another function ? (make sure to fail as early as possible if some other function tries to use the return value despite the error)

Just throwing ideas, let's discuss! What works best ?

Reply:
Yeah I agree returning nil, when an error occurs might be good. Not necessary so.